truffle.js

```diff
@@ -1,51 +1,11 @@
-require('babel-register');
-require('babel-polyfill');
-
-var HDWalletProvider = require('truffle-hdwallet-provider');
-
-const mnemonic = process.env.TEST_MNETONIC || 'burger burger burger burger burger burger burger burger burger burger burger burger';
-const ropstenProvider = new HDWalletProvider(mnemonic, 'https://ropsten.infura.io/');
-const kovanProvider = new HDWalletProvider(mnemonic, 'https://kovan.infura.io');
-
 module.exports = {
   networks: {
-    development: {
+    rpc: {
       network_id: 15,
       host: 'localhost',
       port: 8545,
       gas: 1e8,
-    },
-    test: {
-     provider: require('ethereumjs-testrpc').provider({ gasLimit: 100000000 }),
-     network_id: "*"
-   },
-   development46: {
-     network_id: 15,
-     host: 'localhost',
-     port: 8546,
-     gas: 8e7,
-   },
-   mainnet: {
```

Comment:
Not keeping mainnet as a target? Is the rationale that these contracts won't need to ever be redeployed to mainnet?

Reply:
Yes, and it's unlikely we'll need to interact with the contracts in a console.

We can always add them back later, but regardless, we only need the mined transactions for historical reference.